core/templates/pages/blog-dashboard-page/blog-dashboard-page-root.component.html

```diff
@@ -0,0 +1,105 @@
+<oppia-angular-root>
+  <oppia-base-content>
+    <navbar-pre-logo-action>
+      <oppia-blog-post-editor-pre-logo-action>
+      </oppia-blog-post-editor-pre-logo-action>
+    </navbar-pre-logo-action>
+    <navbar-breadcrumb>
+      <oppia-blog-dashboard-navbar-breadcrumb>
+      </oppia-blog-dashboard-navbar-breadcrumb>
+    </navbar-breadcrumb>
+    <content>
+      <oppia-blog-dashboard-page>
+      </oppia-blog-dashboard-page>
+    </content>
+    <page-footer>
+    </page-footer>
+  </oppia-base-content>
+</oppia-angular-root>
+<style>
+  button:focus {
```

Comment:
Can you please confirm that these are exact copy of the style removed from the other page?